### ISSUE TITLE
Disable webcam from selected user as with microphone

### ISSUE THREAD

**Issue by dakser** [bigbluebutton/bigbluebutton#9995]
Currently we can disable the microphone from a user if needed, we noticed that the same behaviour could be very useful.

What should happen is to be able to disable the active webcam from a selected user.

Thank you for the great BBB.

**Comment by erpsolns** [comment]
Yes I also need.  The situation is like this we have a prayer or bhajan.  At the time of Aarathi user suddenly started video and the whole effort of bhajan is wasted.  If there is disable option for webcam that gives moderator the abilitiy to prevent such situations.

**Comment by unamx88** [comment]
It would be great if the moderator could disable participants' webcams by default in the room settings or even disable webcam transmission for individual participants.

BigBlueButton is brilliant

**Comment by erpsolns** [comment]
Is there any ideas on this needed Enhancement please

**Comment by ffdixon** [comment]
That feature is already in place.  The moderator can lock all viewers from sharing their webcam with the lock settings.

![image](https://user-images.githubusercontent.com/290351/96526945-ef81ae80-124c-11eb-8667-08e625ecba6e.png)


And then lock sharing webcams.

![image](https://user-images.githubusercontent.com/290351/96526961-fc060700-124c-11eb-916c-40a3f3ceeb94.png)

**Comment by dakser** [comment]
Hi Fred, the setting in lock settings is for all users, what's needed is
disable per user as with microphone.

Best regards.
Roberto

**Comment by herzkerl** [comment]
> That feature is already in place. The moderator can lock all viewers from sharing their webcam with the lock settings.
@ffdixon One of our customers asked for this functionality, are you considering changing that or should I tell him to live with the current implementation? TIA!
